devices/milightRF24.coffee

```diff
@@ -78,7 +144,26 @@ module.exports = (env) ->
           button = Buttons.Group4Off
       
       @gateway.sendButton(id, zone, button, false)
-    
+      
+      dataObj = {
```

Comment:
Fixed it now, but there is still a bug, that the white mode is not set on all other zones when you set it in zone 0